### PR TITLE
Fix API method broken in 6e13c3999.

### LINKED ISSUE #1940: Installations from TS broken on dev
probably due to the changes in the API

more details here
https://github.com/galaxyproject/galaxy/pull/1821/files/b15a7ff06df23d7004f670f34638fe9ee5d33f92#r56447368

ping @davebx @bgruening

### PR DESCRIPTION
**Pull request by davebx**
Should fix #1940

### CODE REVIEW

File: lib/galaxy/webapps/tool_shed/api/repositories.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def add_repository_registry_entry( self, trans, payload, **kwd ):
         return response_dict
 
     @web.expose_api_anonymous
-    def get_ordered_installable_revisions( self, trans, **kwd ):
+    def get_ordered_installable_revisions( self, trans, name, owner, **kwd ):
```
Comment (mvdbeek) [inline lib/galaxy/webapps/tool_shed/api/repositories.py]: I guess this should be `get_ordered_installable_revisions( self, trans, name = None, owner = None, **kwd )` ?

Comment (davebx) [inline lib/galaxy/webapps/tool_shed/api/repositories.py]: That syntax wasn't used in the previous working version of this api endpoint, but I don't see a reason _not_ to do it that way.

**Comment by mvdbeek** [comment]
Works for me, :+1:

**Comment by martenson** [comment]
an API test for this method would be splendid but you have my :+1: anyways

**Comment by davebx** [comment]
@martenson That would require a test suite for the toolshed API, which is something we do need (#1944).

**Pull request merged**